curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/64440a50d076c204051f2cc0.md

```diff
@@ -1,46 +1,56 @@
 ---
 id: 64440a50d076c204051f2cc0
-title: Step 34
+title: Step 25
 challengeType: 0
-dashedName: step-34
+dashedName: step-25
 ---
 
 # --description--
 
 You need a function to convert view counts to a more readable format. For example, if the view count is `1000`, it should display as `1k` and if the view count is `100,000` it should display as `100k`.
 
-Create an arrow function called `viewCount` with a parameter called `views`.
+Create a `viewCount` function with a `views` parameter, then return the `views`.
+
+Inside the `viewCount` function, create a `thousands` variable and set it to the value of rounding down `views` based on a thousand.
+
+After that, check if `views` is greater than or equal to `1000`. If it is, use template literals to return the `thousands` variable in the format `${thousands}k`.
```

Comment:
This could be simplified further. 
Basically we should only care that they created the correct function name and are returning the correct value based on the condition.
We shouldn't care if they create a thousands variable or if they use template literals because technically a valid solution could include regular string concatenation with the `+`

something like this could work

```suggestion
Create a `viewCount` function with a `views` parameter. If `views` is greater than or equal to `1000`, return a string with the `views` value divided by `1000` and the letter `k` appended to it. Make sure to round `views / 1000` down to the nearest whole number.

Otherwise, return the `views` value.
```